src/test/java/api/requests/RequestsAPILoanRenewalTests.java

```diff
@@ -455,7 +455,8 @@ public void forbidRenewalOverrideWhenFirstRequestIsNotRecall() {
         "item is not renewable, " +
         "reached number of renewals limit," +
         "renewal date falls outside of the date ranges in the loan policy, " +
-        "items cannot be renewed when there is an active recall request"))));
+        "items cannot be renewed when there is an active recall request, " +
+        "item cannot be renewed: item is Declared lost"))));
```

Comment:
Please can this message be formatted consistently with the other messages

Reply:
The message is according to story's requirement.